### PR TITLE
Update code to include new styles

### PR DESCRIPTION
**Pull request by krisgesling**
A quick update to include
- @intent_handler and intent_file_handler decorators
- showing difference between Adapt and Padatious intents
- highlight the use of `initialize()`
- new `self.log` option
- simpler constructor

Also changed the Marketplace Category from "Daily" to "Configuration", though am thinking that a "Development" category is starting to make sense. 

Any other suggestions welcomed :)

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -16,41 +16,37 @@
...
-__author__ = 'eward'
-
-LOGGER = getLogger(__name__)
-
+from mycroft import MycroftSkill, intent_file_handler, intent_handler
 
 class HelloWorldSkill(MycroftSkill):
     def __init__(self):
-        super(HelloWorldSkill, self).__init__(name="HelloWorldSkill")
+        MycroftSkill.__init__(self)
```
Comment (forslund) [inline __init__.py]: We can skip the entire `__init__` since it doesn't do it or use `super().__init__()`

**Comment by forslund** [comment]
Nice update. We might consider to switch from comments to proper docstrings.

I think the `__init__` should either be removed or explained what sort of initialization should be done there.

**Comment by krisgesling** [comment]
Yeah nice call, does the following make sense?
> The \_\_init\_\_ method is called when the Skill is first constructed. It is often used to declare variables or perform setup actions, however it cannot utilise MycroftSkill methods as the class does not yet exist.

**Comment by forslund** [comment]
Sounds like a good description.

File: __init__.py (hunk 1 above)
Comment (krisgesling) [reply __init__.py]: Kept it and declared a random variable, but switched to super

**Comment by forslund** [comment]
Looks good, I fixed the `super().__init__()` call, shouldn't have `self` as argument.

Merging.

**Pull request merged**